src/fn/compile-app-settings.js

```diff
@@ -39,7 +39,30 @@ const compileAppSettingsForProject = async (projectDir, options) => {
   }
 
   const readOptionalJson = path => fs.exists(path) ? fs.readJson(path) : undefined;
-  const appSettings = fs.readJson(path.join(projectDir, 'app_settings.json'));
+  // Manual configurations should be done in the base-settings.json file
+  // This check and warning can be removed when all project configurations have this new file defined
+  let appSettings = fs.readJson(path.join(projectDir, 'app_settings.json'));
+  const basicSettingsPath = path.join(projectDir, 'app_settings/base-settings.json');
+  if (fs.exists(basicSettingsPath)) {
+    appSettings = fs.readJson(basicSettingsPath);
+  } else {
+    warn(`app_settings.json file should not be edited directly.
+    Please create a basic-settings.json file and move any manually defined configurations there.`);
+  }
+  const formSettings = readOptionalJson(path.join(projectDir, 'app_settings/forms.json'));
+  if (formSettings) {
+    if (appSettings.forms && Object.keys(appSettings.forms).length !== 0) {
+      throw new Error(`forms is defined in both the base settings and the forms.json file.`);
```

Comment:
I'm not sure what to say about this. If the project doesn't have `base_settings.js`, but has `app_settings` and `forms.js`, then the user will get an error every time they try to generate config. 

I think we should gate all these errors in one block, where we test whether app_settings are indeed modularised, and only use these files if they are.

Reply:
The idea is not to have two places that might have conflicting config of the same thing. I like the idea of having one check for modular config.